lib/cc/engine/violation_decorator.rb

```diff
@@ -0,0 +1,41 @@
+module CC
+  module Engine
+    class ViolationDecorator < SimpleDelegator
+      MULTIPLIER_REGEX = %r{\[([\d\.]+)\/([\d\.]+)\]}
+
+      DEFAULT_REMEDIATION_POINTS = 200_000
+
+      def remediation_points
+        points = cop_list.fetch(cop_name, DEFAULT_REMEDIATION_POINTS)
+        if multiplier?
+          points * multiply_by
+        else
+          points
+        end
+      end
+
+      private
+
+      def multiply_by
+        result = message.scan(MULTIPLIER_REGEX)
+        result[0][0].to_i
```

Comment:
It just returns an array. It's super misleading and the alternative is to use named capture groups and `=~` which defines the capture group names as locals.

Not sure I have an opinion on which is better.